packages/mobile-wallet/src/components/Auth/Mnemonic/Confirm/Confirm.js

```diff
@@ -18,31 +20,95 @@ class Confirm extends Component {
     t: PropTypes.func.isRequired,
   };
 
+  state = {
+    isConfirmAlertVisible: false,
+    isConfirmAlertChecked: false,
+
+    isErrorAlertVisible: false,
+  };
+
+  toggleCheckbox = () => {
+    this.setState({ isConfirmAlertChecked: !this.state.isConfirmAlertChecked });
+  };
+
+  onError = () => {
+    this.setState({
+      isConfirmAlertVisible: false,
+      isErrorAlertVisible: false,
+    });
+
+    this.props.onError();
```

Comment:
Let's invoke `onError` function as `setState()` callback - just to be sure that everything is working as expected. I mean something like `this.setState({...}, callback)`